src/Cipher/Caesar/CaesarList.tsx

```diff
@@ -1,4 +1,4 @@
-import * as React from 'react';
+import React, { Component } from 'react';
```

Comment:
Why is React left out of the brackets?

Reply:
React is the name for the whole import, everything in the brackets is a specific export from the module.